modules/src/core/ics02_client/error.rs

```diff
@@ -16,6 +18,270 @@ use crate::signer::SignerError;
 use crate::timestamp::Timestamp;
 use crate::Height;
 
+#[cfg(not(any(test, feature = "ics11_beefy")))]
```

Comment:
unfortunately this macro can't be added inside the `define_error`. That's why I'm forced to duplicate the code. @Wizdave97 if you know of a better way, I'm all ears :)